app-admin/kubelet/kubelet-1.0.2.ebuild

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2015 CoreOS, Inc.. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+# $Header:$
+#
+
+EAPI=5
+
+EGIT_REPO_URI="git://github.com/GoogleCloudPlatform/kubernetes"
+EGIT_COMMIT="e310e619fc1ac4f3238bf5ebe9e7033bf5d47ee2" # v1.0.2
```

Comment:
Alternate idea, use tarballs for fetching from github when we don't really need git itself for live ebuilds and such:

https://github.com/coreos/coreos-overlay/blob/16e4eadad1415a54bd0afce45369932395cf50c1/sys-apps/efunctions/efunctions-1.ebuild

The magic in vcs-snapshot makes sure things work.